guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc

```diff
@@ -5,8 +5,10 @@ You can use the Leapp tool to upgrade as well as to help detect and resolve issu
 
 .Prerequisites
 ifdef::satellite[]
-* Review Known Issues before you begin an upgrade.
-For more information, see {ReleaseNotesDocURL}ref_known-issues_assembly_introducing-red-hat-satellite[Known Issues in {ProjectName} {ProjectVersion}].
+* Review known issues before you begin an upgrade.
+For more information, see {ReleaseNotesDocURL}known-issues[Known Issues in {ProjectName} {ProjectVersion}].
```

Comment:
This line still includes `Known Issues` with upper-case `I`, which doesn't match the linked section heading with lower-case `i`.